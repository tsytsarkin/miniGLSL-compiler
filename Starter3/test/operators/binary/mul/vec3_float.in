{ vec3 a; float b; c = a * b;}
