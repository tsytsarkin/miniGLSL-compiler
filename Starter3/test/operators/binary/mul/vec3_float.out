  1: { vec3 a; float b; c = a * b;}
 (SCOPE (DECLARATIONS (DECLARATION a vec3 (DECLARATION b float))) (STATEMENTS (ASSIGN unknown c (BINARY vec3 * a b))))
